src/lib/rted.rs

```diff
@@ -177,6 +178,56 @@ impl<'a> RTEDConfig<'a> {
             }
         }
     }
+
+    fn set_delta(&mut self, a: usize, b: usize, value: f64, switched: bool) {
+        if switched {
+            self.delta[b][a] = value;
+        } else {
+            self.delta[a][b] = value;
+        }
+    }
+
+    fn set_delta_big(&mut self, a: usize, b: usize, value: u8, switched: bool) {
+        if switched {
+            self.delta_bit[b][a] = value;
+        } else {
+            self.delta_bit[a][b] = value;
+        }
+    }
+
+    fn set_custom_costs(&mut self, cost_del: f64, cost_ins: f64, cost_match: f64) {
+        self.cost_del = cost_del;
+        self.cost_ins = cost_ins;
+        self.cost_match = cost_match;
+    }
+
+    fn set_custom_strategy_vector(&mut self, strategy: Vec<Vec<usize>>) {
+        self.strategy = strategy;
+    }
+
+    fn set_custom_strategy(&mut self, strategy: usize, switched: bool) {
+        self.strategy = vec![];
+        for _ in 0..self.size_src {
+            self.strategy.push(vec![0; self.size_dst]);
+        }
+        if switched {
+            for i in 0..self.size_src {
+                for j in 0..self.size_dst {
+                    if self.itree_src.info[InfoIdx::Post2Size][i]
+                       >= self.itree_dst.info[InfoIdx::Post2Size][j]
+                    {
+                        self.strategy[i][j] = strategy;
+                    } else {
+                        self.strategy[i][j] = strategy + 4;
```

Comment:
Why `+ 4`? As you know, I really don't like magic constants ;)

Reply:
Hrm. I'm not sure what the `4` means, but it looks like this method isn't called in the original code, so I guess we can rm it for now.

Reply:
I have to ask the obvious question: why did we port a function that isn't used? Does that mean our testing isn't catching this sort of thing?

Reply:
The module is so big I'm splitting it into small PRs. This PR just adds some getters and setters that are not tested, but I had presumed they would be called by methods I haven't implemented yet. However, it looks like this one is just dead code in the Java version.

Reply:
When do tests get added? I'm not sure we're gaining much if we merge small PRs that don't have tests.

Reply:
It's a good question. The original code has a very small number of tests which can really only be run when the whole algorithm is complete. For these set and get methods I'm not sure there is much that we can usefully do, but I think we can start creating more useful tests from the next PR onwards.